app/client/components/accountoverview/manageSubscription.tsx

```diff
@@ -225,7 +228,10 @@ export const ManageSubscription = (props: RouteableStepProps) => {
                       </>
                     ) : (
                       <span>FREE</span>
-                    )
+                    ),
+                    fullWidth: productDetail.subscription.payPalEmail
```

Comment:
could this be called `spanTwoCols` or something, just because `fullWidth` is used elsewhere and could be confusing?

Reply:
i see this is done